README.md

```diff
@@ -41,6 +41,29 @@ If you need to run only 1 of the tests you can use
 npm run test:e2e -- --test ./tests/.tmp/e2e/name-of-test-file-name.js
 ```
 
+To run the end-to-end tests, you need to ensure that the following steps have 
+been completed.
+
+1. Goto https://console.firebase.google.com/
+
+2. Choose **sparks-integration-test**
+
+3. Click the cog wheel next to **Overview** and choose **Project Settings**
+
+4. Click tab **SERVICE ACCOUNTS**
+
+5. In Firebase Admin SDK, click **GENERATE NEW PRIVATE KEY**. This will 
+   download a JSON file with the service account settings.
+
+6. Store the file some place safe (outside from your local repo).
+
+7. Store the absolute path the your JSON file in environment variable 
```

Comment:
I think we should switch to using the environment variables that can also be used as documented on Firebase. This will allow us to eventually setup the tests to run on Travis CI